core/Cargo.toml

```diff
@@ -47,6 +47,7 @@ symphonia = { version = "0.5.1", default-features = false, features = ["mp3"], o
 nihav_core = { git = "https://github.com/ruffle-rs/nihav-vp6", rev = "9416fcc9fc8aab8f4681aa9093b42922214abbd3", optional = true }
 nihav_codec_support = { git = "https://github.com/ruffle-rs/nihav-vp6", rev = "9416fcc9fc8aab8f4681aa9093b42922214abbd3", optional = true }
 nihav_duck = { git = "https://github.com/ruffle-rs/nihav-vp6", rev = "9416fcc9fc8aab8f4681aa9093b42922214abbd3", optional = true }
+either = "1.7.0"
```

Comment:
I'd prefer to write this enum ourselves, in my vain effort to reduce our massive amount of dependencies :-)

Reply:
We already have a transitive dependency on `either`. We could try remove it in a follow-up PR, and change the imports added by this PR.

Reply:
+1 for avoiding this dependency, preferably in this PR.

Reply:
If you'd both prefer, I can define `Either` in this PR.